cypress/e2e/signIn.cy.js

```diff
@@ -12,23 +11,26 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
```

Comment:
The `before` hook clears the database before generating a user. Ensure that the `db:clear` task is correctly implemented and does not interfere with the user generation process. Consider verifying the database state after clearing to ensure consistency across tests.